### PR TITLE
ensure cmd use documents positional args

### LINKED ISSUE #167: Registry tool: subcommand usage information should include required positional arguments
Several `registry` subcommands require positional arguments (directories for the bulk upload commands, for example). These currently aren't included in usage information.

Related: we should follow a convention for this. I think `kubectl` is a good guide:

```
Usage:
  kubectl label [--overwrite] (-f FILENAME | TYPE NAME) KEY_1=VAL_1 ... KEY_N=VAL_N [--resource-version=version]
[options]
```

Note that argument values are upper-cased.

### PR DESCRIPTION
**Pull request by theganyo**
Fixes #167

### CODE REVIEW

**Review by timburks — APPROVED** [review]
Thanks! Beyond this, it might be good to put something in [root_test.go](https://github.com/apigee/registry/blob/main/cmd/registry/cmd/root_test.go) to  let us know when new ARGUMENT names appear that aren't in our preferred vocabulary.

File: cmd/registry/cmd/compute/score/score.go [hunk 1]
```diff
@@ -30,7 +30,7 @@ import (
 
 func Command() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "score",
+		Use:   "score RESOURCE",
```
Comment (timburks) [inline cmd/registry/cmd/compute/score/score.go:33]: Maybe this should be PATTERN, but we can revisit that.

Comment (theganyo) [reply cmd/registry/cmd/compute/score/score.go:33]: No, you're right, PATTERN is correct.

**Comment by theganyo** [comment]
> Thanks! Beyond this, it might be good to put something in [root_test.go](https://github.com/apigee/registry/blob/main/cmd/registry/cmd/root_test.go) to let us know when new ARGUMENT names appear that aren't in our preferred vocabulary.

This seem non-trivial, I'll add a new issue for it.

**Pull request merged**